tools/codegen/gen.py

```diff
@@ -429,13 +484,48 @@ def generate_defn(faithful: bool) -> str:
     return at::_ops::{f.func.name.unambiguous_name()}::redispatch({exprs_str});
 }}
 """
+
         result = generate_defn(False)
         if sig_group.faithful_signature is not None:
             result += generate_defn(True)
 
         return result
 
 
+# Generates Static.cpp.
+# This generates static unboxing wrapper for ATen ops.
+@dataclass(frozen=True)
+class ComputeUnboxingWrapper:
```

Comment:
Similarly, can we move all logic in this file relevant to creating `CodegenUnboxingWrapper.cpp` into a separate file, that way it's all independent of aten?

It will need to re-use some helper functions in this file to create the list of `NativeFunctions` objects, but otherwise I think it can be pretty independent.

@albanD is `tools/autograd` a reasonable home for `torch::jit`-related codegen? It's probably bettern than this file, which only handles `aten` code. Or maybe we could make a `tools/jit` folder, if `tools/autograd` isn't a great fit.

Reply:
I think `tools/jit` is a better place.
Having it in `tools/autograd` makes the codeowner a lot more noisy.